pkgs/dart_services/lib/src/common_server.dart

```diff
@@ -115,6 +121,52 @@ class CommonServerApi {
     return ok(version().toJson());
   }
 
+  /// Handle a new websocket connection request.
+  ///
+  /// Handle incoming requests, convert them to exising commands and dispatch
+  /// them appropriately. The commands and responses mirror the existing REST
+  /// protocol.
+  ///
+  /// This will be a long-running conneciton to the client.
+  void handleWebSocket(WebSocketChannel webSocket, String? subprotocol) {
+    webSocket.stream.listen(
```

Comment:
It's probably a good idea to store the returned StreamSubscription object and clean it up in the onDone callback to avoid a memory leak

Reply:
👍 , will do